### ISSUE TITLE
Control-[left/right] keyboard navigation doesn't work

### ISSUE THREAD

**Issue by lgarron** [nushell/nushell#9292]
### Describe the bug

Per https://www.nushell.sh/blog/2020-01-07-nushell-0_8_0.html#improved-keyboard-navigation-aloso:

> We now support moving left by-word using the ctrl+left and ctrl+right keyboard shortcuts.

This does not work for me on macOS Ventura 13.4 using any of:

- `Terminal.app`
- iTerm 3.4.19
- VS Code terminal

Using ⌘⬅️ or ⌘➡️ does not work either, and neither do a few other alternatives I've tried.

### How to reproduce

1. Open a `nu` shell on macOS
2. Type a command with multiple words and press Ctrl-left.

### Expected behavior

The cursor moves a word to the left.

### Configuration

| key                | value                                                             |
| ------------------ | ----------------------------------------------------------------- |
| version            | 0.80.0                                                            |
| branch             |                                                                   |
| commit_hash        |                                                                   |
| build_os           | macos-aarch64                                                     |
| build_target       | aarch64-apple-darwin                                              |
| rust_version       | rustc 1.69.0 (84c898d65 2023-04-16) (built from a source tarball) |
| cargo_version      | cargo 1.69.0 (6e9a83356 2023-04-12)                               |
| build_time         | 2023-05-16 22:11:13 +00:00                                        |
| build_rust_channel | release                                                           |
| features           | default, zip                                                      |
| installed_plugins  |                                                                   |

**Comment by fdncred** [comment]
I use `option-left` `option-right` on macos without issue. It's not surprising that a 3 year old link would have outdated information. I believe I'm using these keybindings to do that.
```
  { 
    name: complete_hint_chunk
    modifier: alt
    keycode: right
    mode: emacs
    event: { until: [ 
      { send: historyhintwordcomplete }
      { edit: movewordright }
    ]
    }
  }
  { 
    name: un_complete_hint_chunk
    modifier: alt
    keycode: left
    mode: emacs
    event: [ 
      { edit: backspaceword }
    ]
  }
```

The key on MacOS is to make sure your terminal support the keybindings because MacOS is just weird. You can test this with the `keybindings listen` command.

**Comment by lgarron** [comment]
> ```
> 
> ```
> 
> 
>     
>   
> 
> The key on MacOS is to make sure your terminal support the keybindings because MacOS is just weird. You can test this with the `keybindings listen` command.

Thanks! `keybindings listen` seems to swallow every modifier key for the arrow keys except `SHIFT` in iTerm due to a workaround I need for other shells, but ⌥⬅️ and ⌥➡️ do work in `Terminal.app`!

**Comment by fdncred** [comment]
I got it to work in iterm2 but i'm not on my mac right now. there's definitely a setting in the app to control it.
